src/components/Audio.tsx

```diff
@@ -0,0 +1,25 @@
+import { useGlobalContext } from "../context/Global";
+
+const Audio = () => {
```

Comment:
```suggestion
const Audio = () => {
```

Give this a proper name. Like `SwapSuccessSoundSetting`